src/org/zaproxy/zap/extension/formhandler/resources/Messages.properties

```diff
@@ -19,4 +19,5 @@ formhandler.options.dialog.field.remove.title = Remove A Field
 formhandler.options.dialog.field.remove.text = Are you sure you want to remove the selected field.
 formhandler.options.dialog.field.remove.checkbox.label = Do not show this message again
 formhandler.options.dialog.field.remove.button.confirm = Remove
-formhandler.options.dialog.field.remove.button.cancel = Cancel
\ No newline at end of file
+formhandler.options.dialog.field.remove.button.cancel = Cancel
+formhandler.popup.menu.params.add.label = Add as Form Handler Parameter
```

Comment:
I'm not totally sold on this text, if anyone has ideas please share.

Reply:
`Add Form Handler Field`?

Reply:
Sounds good to me. Tweak inbound.